agent/src/index.ts

```diff
@@ -495,6 +473,18 @@ export async function createAgent(
         );
     }
```

Comment:
We can nuke this goatPlugin i don't think its used

Reply:
nuked: https://github.com/elizaOS/eliza/pull/1383/commits/12d8537c179f2201d1d3eb5da9afd39cdd3f3ba9